waku/waku_archive/common.nim

```diff
@@ -18,14 +19,16 @@ import
 
 type MessageDigest* = MDigest[256]
 
-proc computeDigest*(msg: WakuMessage): MessageDigest =
+proc computeDigest*(msg: WakuMessage, pubSubTopic: string = DefaultPubsubTopic): MessageDigest =
```

Comment:
Since every message would come through a pubSub topic, what is the point of the default here? (Feels like it would only lead to issues in the future if we forget to pass in a pubSub topic and then get inconsistent behaviour)

Reply:
Yes, but the pubSub topic needs to be passed to the `computeDigest` function as it is not a WakuMessage attribute.

The default is used since we are using the existing function in so many test cases already, one way to get rid of that (without changing all the other test cases) is:
- extend the function with a default (what I've used) or,
- add another function with some sort of similar name and redirect it from the queue driver and Waku archive code for production workflow

better of the two seems to be the former. WDYT @vpavlin?

Reply:
Why is "update the tests" not an option?:)